packages/kuma-gui/src/test-support/mocks/src/meshes/_/dataplanes/_overview.ts

```diff
@@ -101,7 +101,8 @@ export default ({ fake, pager, env }: EndpointDependencies): MockResponder => (r
                 outbound: Array.from({ length: outboundCount }).map((_, _i) => {
                   return {
                     port: fake.internet.port(),
-                    tags: fake.kuma.tags({ service }),
+                    tags: fake.kuma.tags({ service: fake.word.noun() }),
+                    address: fake.internet.ipv4(),
```

Comment:
If address and port are potentially undefined, should we reproduce that here?

Reply:
Oh also, can you only use ipv4 for outbounds? I would have thought these can be ipv6 also?